### ISSUE TITLE
Add admin and review links for add-on details page

### ISSUE THREAD

**Issue by jvillalobos** [mozilla/addons-frontend#4632]
Broken off from #2756.
~~Depends on #5548~~

These links should appear in the "More Information" box, at the bottom of the "Add-on Links" section. The following links are needed:

* "Edit add-on". Goes to `/developers/addon/SLUG/edit`. Requires `Addons:Edit` permission to appear.
* "Admin add-on". Goes to `/admin/addon/manage/SLUG/`. Requires `AdminTools:View` permission to appear. 
* "Content review add-on". Goes to `/reviewers/review-content/SLUG`. Requires  `Addons:ContentReview` permission to appear.
* "Code review add-on". Goes to `/reviewers/review/SLUG`. Requires `Addons:PostReview` permission to appear. Should not appear for themes.
* "Review theme". Goes to `/reviewers/review/SLUG`. Requires `Personas:Review` permission to appear. Should only appear for themes.

**Comment by jvillalobos** [comment]
@caitmuenster This is still relatively low priority, but maybe you can lobby for someone to help you with it sooner than later.

**Comment by wagnerand** [comment]
@jvillalobos I know I'm late, but I am wondering if it would be better to 

* Rename `Admin add-on` to `Admin add-on status`
* Add "Admin add-on". Goes to `admin/models/addons/addon/ID`. Requires `AdminTools:View` and `Addons:Edit` permission to appear.

**Comment by jvillalobos** [comment]
Fine with me if it's okay with @bobsilverberg

**Comment by bobsilverberg** [comment]
I haven't landed the changes yet so yes, I'm fine to make these changes.

**Comment by ioanarusiczki** [comment]
@bobsilverberg Hi Bob, here are my test results on AMO dev with FF62-Win10 with three questions:

1.  Admin tools -  The "Admin add-on" link has the following slug: admin/models/addons/addon/ID/change/
and there is also "Admin add-on status" link with this slug: /admin/addon/manage/SLUG/ (that's the one mentioned by @jvillalobos above.  I don't see an issue here, but I'd like you to confirm it's alright.

2.  Approved static themes ->  for Admins and Content Reviewers there is "Content review add-on" link which opens a page such as this one https://reviewers.addons-dev.allizom.org/en-US/reviewers/review-content/rm-static-theme-3.  
Admins and Theme Reviewers can access " Review theme" meaning this page
https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/rm-static-theme-3

So, I've noticed that the [Content review](https://reviewers.addons-dev.allizom.org/en-US/reviewers/queue/content_review) queue  from Rev Tools does not contain static themes in the list.  Is the "Content review add-on" link and "Review theme" still necessary for approved static themes so that the reviewer can still access this easier and reply to a developer ? 

3.  Approved lw themes -> An Admin and Content reviewer have the "Content review add-on" link such as this example https://reviewers.addons-dev.allizom.org/en-US/reviewers/review-content/three-wolf-moon-shirt where a user can only add a comment. If I go to  Rev Tools I do not see a content review queue for lw themes. Is this link and page opened of any use?

Another thing I've noticed for lw themes is that a user from "Reviewers:Themes" group, when "Review theme" is clicked the page in not displayed. Let me know if I must file a separate issue for this one.

![lw theme clicking the link](https://user-images.githubusercontent.com/33448286/46609889-f0d25980-cb11-11e8-92a6-4b43bbf7bd96.gif)

**Comment by bobsilverberg** [comment]
@ioanarusiczki Thanks for checking this out. There is one thing you mentioned that confuses me:

> Admin tools - The "Admin add-on" link has the following slug: admin/models/addons/addon/ID/change/.

What's the `/change/` at the end of that URL? When I access dev I don't see that, and it's not in the code. That should not follow the `ID` on the URL. Can you please double check that? Other than that, what you've stated in #1 above is what is expected.

As for your other observations, I'm not particularly knowledgeable in how all this is supposed to work - I just made sure to create the links based on the requirements described above.

@wagnerand could you please take a look at @ioanarusiczki's other comments above and verify whether there are any issues with the code as it now stands, or whether any follow-up issues need to be filed to address any of it?

**Comment by wagnerand** [comment]
> So, I've noticed that the Content review queue from Rev Tools does not contain static themes in the list. Is the "Content review add-on" link and "Review theme" still necessary for approved static themes so that the reviewer can still access this easier and reply to a developer ?

I'll forward to @caitmuenster on whether we should do content-review for themes or whether the current theme-review process is enough.

> Approved lw themes -> An Admin and Content reviewer have the "Content review add-on" link such as this example reviewers.addons-dev.allizom.org/en-US/reviewers/review-content/three-wolf-moon-shirt where a user can only add a comment. If I go to Rev Tools I do not see a content review queue for lw themes. Is this link and page opened of any use?

Same here, although lightweight themes are getting migrated to new themes very soon, so I wouldn't worry too much about this. Imho, we can just ignore this case.

> Another thing I've noticed for lw themes is that a user from "Reviewers:Themes" group, when "Review theme" is clicked the page in not displayed. Let me know if I must file a separate issue for this one.

I am not sure whether lightweight themes are gated by that permission, although I think they are. I'd say file a follow-up issue and let's figure it out there.

> What's the /change/ at the end of that URL? When I access dev I don't see that, and it's not in the code. That should not follow the ID on the URL. Can you please double check that?

That is added by django admin as far as I can tell, so it should be ok.

**Comment by caitmuenster** [comment]
Thanks @wagnerand! 

The current theme review process is enough, no need for content review for themes.

**Comment by wagnerand** [comment]
> The current theme review process is enough, no need for content review for themes.

Then themes shouldn't show a "content review add-on" button.

**Comment by bobsilverberg** [comment]
> Then themes shouldn't show a "content review add-on" button.

Thanks @wagnerand. I opened https://github.com/mozilla/addons-frontend/issues/6559 as a follow-up.

**Comment by wagnerand** [comment]
Thanks @bobsilverberg.

> Another thing I've noticed for lw themes is that a user from "Reviewers:Themes" group, when "Review theme" is clicked the page in not displayed. Let me know if I must file a separate issue for this one.
> I am not sure whether lightweight themes are gated by that permission, although I think they are. I'd say file a follow-up issue and let's figure it out there.

@ioanarusiczki could you file a follow-up for this, please?

**Comment by ioanarusiczki** [comment]
@wagnerand Sure. I'll return to this issue asap and file all it's needed.

**Comment by ioanarusiczki** [comment]
@bobsilverberg About the "Admin add-on" link, I can confirm again that it goes to this slug https://addons-internal.dev.mozaws.net/en-US/admin/models/addons/addon/ID/change/
But I think it should be ok , it's from django admin as @wagnerand mentioned [here](https://github.com/mozilla/addons-frontend/issues/4632#issuecomment-428177323)

Here's an example 

![change](https://user-images.githubusercontent.com/33448286/46723488-a8857980-cc80-11e8-9794-5d37d70b3dcc.gif)

**Comment by ioanarusiczki** [comment]
@bobsilverberg Ok,thanks !
 I'm marking this verified fixed on AMO dev - FF62(Win10).

**Comment by ioanarusiczki** [comment]
@bobsilverberg Sorry for noticing so late, but while re-checking the links today during release testing I've also noticed this issue https://github.com/mozilla/addons-frontend/issues/6590 for lw themes.
